packages/_private/vue-demo/src/components/ThemeSwitch.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+import { SynVueButton } from '@synergy-design-system/vue';
+import { SynButton as SynButtonElement } from '@synergy-design-system/components';
+
+const switchTheme = (e: Event) => {
+  const bdy = document.body;
```

Comment:
See comment for https://github.com/synergy-design-system/synergy-design-system/pull/238/files#diff-ab22fd38a77da3408a668fd2133027984d431c3eef05093a33705bae0e3f87ccR13